liquibase-core/src/test/groovy/liquibase/command/InternalHistoryCommandStepTest.groovy

```diff
@@ -0,0 +1,71 @@
+package liquibase.command
+
+import liquibase.change.CheckSum
+import liquibase.changelog.ChangeLogHistoryService
+import liquibase.changelog.ChangeLogHistoryServiceFactory
+import liquibase.changelog.ChangeSet
+import liquibase.changelog.RanChangeSet
+import liquibase.command.core.InternalHistoryCommandStep
+import liquibase.database.Database
+import liquibase.database.DatabaseConnection
+import spock.lang.Specification
+
+import java.nio.charset.StandardCharsets
+
+import static liquibase.servicelocator.PrioritizedService.PRIORITY_DATABASE
+
+class InternalHistoryCommandStepTest extends Specification {
+
+    def "displays the history in tables"() {
+        given:
+        def changeLogHistoryService = Mock(ChangeLogHistoryService)
+        changeLogHistoryService.getRanChangeSets() >> [
+                new RanChangeSet("some/change/log", "some/id", "me", CheckSum.compute("foobar"), new Date(1670000000000), "", ChangeSet.ExecType.EXECUTED, "", "", null, null, "deployment-id-1"),
+                new RanChangeSet("some/change/log", "some/other/id", "me", CheckSum.compute("foobar"), new Date(1670000000000), "", ChangeSet.ExecType.EXECUTED, "", "", null, null, "deployment-id-1"),
+                new RanChangeSet("some/change/log", "yet/another/id", "me", CheckSum.compute("foobar"), new Date(1675000000000), "", ChangeSet.ExecType.EXECUTED, "", "", null, null, "deployment-id-2")
+        ]
+        def database = databaseAt("jdbc:some://url")
+        changeLogHistoryService.supports(database) >> true
+        changeLogHistoryService.getPriority() >> PRIORITY_DATABASE
+        ChangeLogHistoryServiceFactory.getInstance().register(changeLogHistoryService)
+        def step = new InternalHistoryCommandStep()
+        def outputStream = new ByteArrayOutputStream()
+        def command = new CommandScope("history")
+                .addArgumentValue("database", database)
+                .addArgumentValue("format", "TABULAR")
+        def builder = new CommandResultsBuilder(command, outputStream)
+
+        when:
+        step.run(builder)
+
+        then:
+        def output = new String(outputStream.toByteArray(), StandardCharsets.UTF_8)
+        output.trim() == """
+Liquibase History for jdbc:some://url
+
++-----------------+------------------+-----------------+-------------------+---------------+------------------------------------+
```

Comment:
This test is failing  as this output is not equal to what it's being returned by `output.trim()`

Reply:
Problem with timezones, cheated a bit and went with just displaying the year (see 008944bb90519291a595a9fd0bee48000e530e74)